### ISSUE TITLE
TorchInductor OpInfo accuracy failure Tracker

### ISSUE THREAD

**Issue by SherlockNoMad** [pytorch/torchdynamo#1359]
Following OpInfo tests are failing due to accuracy issue:

To reproduce the error, run 
```
pytest -sv  test/test_torchinductor_opinfo.py -k test_comprehensive_<opname>_<device_type>_<dtype>
```

cc @anijain2305 

- [ ] _masked.argmax	cuda	torch.float16	
- [ ] _masked.argmax	cuda	torch.float32	
- [ ] _masked.argmax	cuda	torch.float64	
- [ ] _masked.argmax	cuda	torch.int32	
- [ ] _masked.argmax	cuda	torch.int64	
- [ ] _masked.argmin	cuda	torch.float16	
- [ ] _masked.argmin	cuda	torch.float32	
- [ ] _masked.argmin	cuda	torch.float64	
- [ ] _masked.argmin	cuda	torch.int32	
- [ ] _masked.argmin	cuda	torch.int64	
- [ ] _masked.cumprod	cuda	torch.float16	
- [ ] _masked.cumsum	cuda	torch.float16	
- [ ] add	cuda	torch.bool	
- [ ] addbmm	cuda	torch.float16	
- [ ] addmv	cuda	torch.float16	
- [ ] addr	cuda	torch.float16	
- [ ] argmax	cuda	torch.float16	
- [ ] argmax	cuda	torch.float32	
- [ ] argmax	cuda	torch.float64	
- [ ] argmax	cuda	torch.int32	
- [ ] argmax	cuda	torch.int64	
- [ ] argmin	cuda	torch.float16	
- [ ] argmin	cuda	torch.float32	
- [ ] argmin	cuda	torch.float64	
- [ ] argmin	cuda	torch.int32	
- [ ] argmin	cuda	torch.int64	
- [ ] baddbmm	cuda	torch.float16	 
- [ ] cross	cuda	torch.float16	
- [ ] cumsum	cuda	torch.float16	
- [ ] diff	cuda	torch.float16	
- [ ] dist	cuda	torch.float16	
- [ ] linalg.cholesky	cuda	torch.float32	
- [ ] linalg.cholesky	cuda	torch.float64	
- [ ] linalg.cholesky_ex	cuda	torch.float32	
- [ ] linalg.cholesky_ex	cuda	torch.float64	
- [ ] linalg.cross	cuda	torch.float16	
- [ ] linalg.lu_factor	cuda	torch.float32	
- [ ] linalg.lu_factor	cuda	torch.float64	
- [ ] linalg.lu_factor_ex	cuda	torch.float32	
- [ ] linalg.lu_factor_ex	cuda	torch.float64	
- [ ] logcumsumexp	cuda	torch.float16	
- [ ] logical_and	cuda	torch.int32	
- [ ] logical_and	cuda	torch.int64	
- [ ] logical_or	cuda	torch.int32	
- [ ] logical_or	cuda	torch.int64	
- [ ] matrix_exp	cuda	torch.float16	
- [ ] max.reduction_with_dim	cuda	torch.int32	
- [ ] max.reduction_with_dim	cuda	torch.int64	
- [ ] min.reduction_with_dim	cuda	torch.int32	
- [ ] min.reduction_with_dim	cuda	torch.int64	
- [ ] nn.functional.binary_cross_entropy	cuda	torch.float16	
- [ ] nn.functional.multi_margin_loss	cuda	torch.float16	
- [ ] nn.functional.multilabel_margin_loss	cuda	torch.float16	
- [ ] nn.functional.soft_margin_loss	cuda	torch.float16	
- [ ] renorm	cuda	torch.float16	
- [ ] select_scatter	cuda	torch.int32	
- [ ] select_scatter	cuda	torch.int64	
- [ ] sign	cuda	torch.bool	
- [ ] sign	cuda	torch.int32	
- [ ] sign	cuda	torch.int64

**Comment by voznesenskym** [comment]
As long as cuda ops on triton leave behind bad state, these burn down lists wont be useful.

**Comment by SherlockNoMad** [comment]
> As long as cuda ops on triton leave behind bad state, these burn down lists wont be useful.

When cuda ops get over the inductor and triton failures, they eventually get to this list, no ?

**Comment by jansel** [comment]
If we disable cudagraphs does things improve?

**Comment by eellison** [comment]
We shouldn't be testing accuracy for the empty ops. Removing them from the list.

**Comment by SherlockNoMad** [comment]
Closing this issue, as we track all the failures in this issue https://github.com/pytorch/torchdynamo/issues/1422

**Comment by desertfire** [comment]
I actually prefer to keep this open as these probably have a higher priority, given the urgency of tackling model accuracy errors.

**Comment by desertfire** [comment]
The `argmax/argmin` ops and those implemented built on top of them have this know issue, https://github.com/pytorch/torchdynamo/issues/859. Basically when there are multiple same `max/min` values in a tensor, it is hard to force triton returns the same deterministic index as eager.

I can imagine adding a flag to force fallback ops in the accuracy mode, but I also have no idea how these ops are used in real world and does determinism matter here? cc @Chillee @ngimel

**Comment by ngimel** [comment]
#1444 - 1-element argmin/argmax fix
